docs/docs/intro.md

```diff
@@ -9,11 +9,12 @@ firmware built on the [Zephyr™ Project](https://zephyrproject.org/) Real Time
 
 ## Features
 
-At this point, ZMK is still missing many features compared to popular firmware such as QMK. Here is a table comparing supported ZMK and QMK features:
+At this point, ZMK is still missing many features compared to popular firmware such as QMK. Here is a table comparing supported ZMK, BlueMicro, and QMK features:
```

Comment:
Now that we've added a BM column, perhaps this is more appropriate:
`ZMK is currently missing some features found in other popular firmware.  This table compares the features supported by ZMK, BlueMicro and QMK:`